test/KoreBuild.FunctionalTests/KoreBuild.FunctionalTests.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+<Import Project="..\..\build\common.props" />
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.CommandLineUtils.Sources" Version="$(AspNetCoreVersion)" PrivateAssets="All" />
```

Comment:
That's a good observation, and one I plan to address in a follow-up as a part of the "version pinning" work. For now, this repo already has a bunch of dependencies on aspnetcore packages, so it will be a big change that will affect existing tools.